### ISSUE TITLE
Menu focus does not return on anchor element when closed

### ISSUE THREAD

**Issue by mimarz** [equinor/design-system#1704]
When `Menu` is closed when using keyboard navigation, the anchor element should be focused or next in line for focus according to https://www.w3.org/TR/wai-aria-practices-1.1/#menu
